OGO/ogo-shield/_meta/manifest.yml

```
@@ -0,0 +1,4 @@
+uuid: cf5c916e-fa26-11ed-a844-f7f4d7348199
+name: OGO Shield
+slug: ogo-shield
+description: OGO Shield Web Application Firewall provides protection and performance for your web applications and APIs.
```

A datasource for the format.
```suggestion
description: OGO Shield Web Application Firewall provides protection and performance for your web applications and APIs.

data_sources:
  Web application firewall logs: OGO protects web applications and APIs
```